### PR TITLE
Change https://github.com/google/gopacket to https://github.com/gopac…

### LINKED ISSUE #34536: Switch from google/gopacket to gopacket/gopacket
This version of the library seems to be more well-supported by the community and gets regular updates:
https://github.com/gopacket/gopacket

### PR DESCRIPTION
**Pull request by crashiura**
Switch from google/gopacket to gopacket/gopacket

Fixes: #34536

### CODE REVIEW

File: pkg/hubble/parser/threefour/parser.go [hunk 1]
```diff
@@ -157,7 +157,7 @@ func (p *Parser) Decode(data []byte, decoded *pb.Flow) error {
 	defer p.packet.Unlock()
 
 	// Since v1.1.18, DecodeLayers returns a non-nil error for an empty packet, see
-	// https://github.com/google/gopacket/issues/846
+	// https://github.com/gopacket/gopacket/issues/846
```
Comment (tklauser) [inline pkg/hubble/parser/threefour/parser.go]: I think this should be left as is, as it is referring to an issue on the original repository that doesn't exist on the new repository.

Comment (crashiura) [reply pkg/hubble/parser/threefour/parser.go]: yes, thank you

**Review by tklauser — CHANGES_REQUESTED** [review]
@crashiura could you please squash the two commits? Otherwise LGTM, thank you!

**Review by joestringer — APPROVED** [review]
Looks good, thanks!

**Review by tklauser — APPROVED** [review]

**Review by marseel — APPROVED** [review]
Thanks!

**Review by rgo3 — APPROVED** [review]

**Review by kaworu — APPROVED** [review]
Thank you @crashiura !

**Pull request merged**